src/Tools/Microsoft.CodeAnalysis.Testing/Microsoft.CodeAnalysis.Analyzer.Testing/Microsoft.CodeAnalysis.Analyzer.Testing.csproj

```diff
@@ -0,0 +1,21 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <TargetFrameworks>netstandard1.3;net461</TargetFrameworks>
+    <GenerateDocumentationFile>true</GenerateDocumentationFile>
+    <LangVersion>7.3</LangVersion>
+    <AssemblyName>Microsoft.CodeAnalysis.Analyzer.Testing</AssemblyName>
+    <RootNamespace>Microsoft.CodeAnalysis.Analyzer.Testing</RootNamespace>
+  </PropertyGroup>
+  
+  <PropertyGroup>
+    <IsPackable>true</IsPackable>
+    <Description>Roslyn Analyzer Test Framework Common Types.</Description>
+    <PackageReleaseNotes>Roslyn Analyzer Test Framework Common Types</PackageReleaseNotes>
+    <PackageTags>Roslyn Analyzer Test Framework Common</PackageTags>
+  </PropertyGroup>
+
+  <ItemGroup>
+    <PackageReference Include="Microsoft.CodeAnalysis.Workspaces.Common" Version="2.9.0" />
```

Comment:
:exclamation: needs to be 1.2.1.

Reply:
Do we need to also release a package based on 2.6.0 to enable testing IOperation analyzers?

Reply:
@sharwell 1.0.1 is the oldest version for which nuget dependencies are correct (1.0.0 got this wrong)

@mavasani This points out an important problem.  We could release mirrored packages such that, for whatever version of roslyn the user is targeting, an equivalent testing library exists.

Reply:
@mavasani what other analyzer features have we shipped and in which versions?

| Version | Feature             |
| :-----: | :-----------------: |
| 2.0?    | Concurrent analysis |
| 2.6     | IOperation          |

Reply:
> Concurrent analysis

1.2.x

> IOperation

2.x, but the test framework doesn't need to know about it. StyleCop Analyzers uses assembly binding redirection to test things that require 2.x even though the core of the test harness references 1.2.1.